### PR TITLE
discussion template

### PR DESCRIPTION
**Pull request by JustinZhengBC**
i changed something in global.css to allow scrolling

### CODE REVIEW

**Comment by tomyang729** [comment]
Good job overall!

I found a few bugs when I test it on my browser:
1. Unhandled error showing in the console. (somthing like 'title of undefined')
2. The matCard component seems like not having responsive styling. (If you type in a very long text, and then shrink the screen, the text are out of box)
3. The matDialogContainer modal should also have responsive width.

@omzmarlon

**Comment by omzmarlon** [comment]
@JustinZhengBC I've just got some time to look at yours and Madeleine's code. Since you guys code are similar,  take a look at my comments in Madeleine's PR and then apply similar changes to your code so that I don't have to repeat myself : )

**Comment by JustinZhengBC** [comment]
I fixed the resizing issues and got rid of the "title of undefined" error that happens when you cancel a new post instead of submitting it. I also reorganized code according to the comments in Madeleine's PR where appropriate.

File: web/src/redcross/discussionModule/components/DiscussionComponent.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export class DiscussionComponent {
     private posts: Post[];
     private currentUser: Author;
 
-    constructor(public dds: DiscussionDataService,
-                public md: MatDialog
+    constructor(private dds: DiscussionDataService,
```
Comment (omzmarlon) [inline web/src/redcross/discussionModule/components/DiscussionComponent.ts]: Rename this component to DiscussionContainerComponent. Also rename the html and less

Also move the AddPostDialog class into a separate file

**Review by omzmarlon — CHANGES_REQUESTED** [review]
Also use nesting in .less files to better reflect structures in html

**Comment by JustinZhengBC** [comment]
Not sure if I did the nesting properly, but lint doesn't have any problems with it

**Review by omzmarlon — APPROVED** [review]

**Comment by omzmarlon** [comment]
Good job!

**Pull request merged**